src/main/java/io/codelex/enums/practice/exercise_1.java

```diff
@@ -0,0 +1,33 @@
+package io.codelex.enums.practice;
+
+public class exercise_1 {
```

Comment:
`Exercise1`

Reply:
Nomainīju. Intereses pēc kādēļ nevajadzētu rakstīt exercise_2, bet jāraksta exercise2?

Reply:
ar lielo burtu arī 😄 ir tādi vispārīgi pieņemti noteikumi priekš formatēšanas, katrai valodai savi 😄 
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
Tad jau man viss slikti bijis ar virsrakstiem, mēģināšu ievērot ;)